src/Types.jl

```diff
@@ -140,14 +142,17 @@ end
 
 const VersionTypes = Union{VersionNumber,VersionSpec,UpgradeLevel}
 
-# The url field can also be a local path, rename?
 Base.@kwdef mutable struct GitRepo
-    url::Union{Nothing,String} = nothing
+    source::Union{Nothing,String} = nothing
```

Comment:
I wonder if `remote` instead of `source` is a better name. It is the remote which can both be a url and a local path.

Reply:
Yeah, `remote` is a better name. `Pkg.dependencies()` already uses `source` to mean something else so it might get confusing.

Reply:
I thought about this a bit more and perhaps it is not such a good name after all. If you have a repo on your local disc, the path to that repo is not the remote of the repo. There can also be multiple remotes of a repo etc... Hmm.